plugins/node/opentelemetry-instrumentation-net/src/types.ts

```diff
@@ -28,4 +28,17 @@ export enum SocketEvent {
   CLOSE = 'close',
   CONNECT = 'connect',
   ERROR = 'error',
+  SECURE_CONNECT = 'secureConnect',
+}
+
+export enum TLSAttributes {
```

Comment:
Are any of these in the Semantic Conventions?

Reply:
Not that I am aware of. I checked the specs and didn't find them there

Reply:
Sorry, I was confused. I thought you talk about the socket events, but you mean the TLSAttributes? I actually opened an issue already: https://github.com/open-telemetry/opentelemetry-specification/issues/1652

> can you add a comment that these are not "official"

Yes, I can do that.

I guess they are not the only candidates for that, e.g. the DNS package puts the name that was looked up into net.peer.name, which is probably not correct and their _might_ be a semantic convention for DNS.

Since this is kind of related to https://github.com/open-telemetry/opentelemetry-js/issues/2123: Would it help if I scan js and js-contrib (semi)automatically for attributes that are either in the semantic-specs already or that might be candidates for it?

Reply:
that would be great

Reply:
@dyladan I found a bunch of them. There's four different cases:

1) Use of existing semantic conventions (like PG, see #467)
2) Use of potentially new semantic conventions, I mean things that are used outside of node.js as well (DNS, TLS, graphQL)
3) Use of custom attributes specific to the instrumented library (express, hapi, restify)
4) A corner case are browser specific attributes: while those are "unique" to javascript I would still argue that they should be covered by a semantic convention, since they are still universal

Since this is a discussion unrelated to this ticket, can you give me some guidance where to move this discussion? Open another issue with the specification repo?

Reply:
I would open an issue on this repo to track it at least. If you think some should be spec'd then you should open an issue on the spec repo too and link them.